CHANGES.md

```diff
@@ -1,9 +1,35 @@
-## In the next release...
+## 1.2.0 2017-09-07
 
 * **Breaking Change**: `io.l5d.mesh`, `io.l5d.thriftNameInterpreter`, linkerd
   admin, and namerd admin now serve on 127.0.0.1 by default (instead of
   0.0.0.0).
-* **Breaking Change**: Removed support for PKCS#1 keys.
+* **Breaking Change**: Removed support for PKCS#1-formatted keys. PKCS#1 formatted keys must be converted to PKCS#8 format.
+* Added experimental `io.l5d.dnssrv` namer for DNS SRV records (#1611)
+* Kubernetes
+  * Added an experimental `io.l5d.k8s.configMap` interpreter for reading dtabs from a Kubernetes ConfigMap (#1603). This interpreter will respond to changes in the ConfigMap, allowing for dynamic dtab updates without the need to run Namerd.
+  * Made ingress controller's ingress class annotation configurable (#1584).
+  * Fixed an issue where Linkerd would continue routing traffic to endpoints of a service after that service was removed (#1622).
+  * Major refactoring and performance improvements to `io.l5d.k8s` and `io.l5d.k8s.ns` namers (#1603).
+  * Ingress controller now checks all available ingress resources before using a default backend (#1607).
+  * Ingress controller now correctly routes requests with host headers that contain ports (#1607).
+* HTTP/2
+  * Fixed an issue where long-running H2 streams would eventually stop accepting new messages (#1598).
```

Comment:
if I'm being _reallllllly_ nitpicky, a long-running stream should never accept new messages, only new frames.  :neckbeard: 

We could also probably use less technical terminology: "... eventually hang."